layouts/default.vue

```diff
@@ -1,6 +1,17 @@
 <template>
   <v-app id="app">
-    <SbcHeader class="sbc-header" />
+    <SbcHeader class="sbc-header" :in-auth="false" />
```

Comment:
Since `inAuth` prop already has a default of False, I think it's better to not include it here.